release.json

```diff
@@ -1,22 +1,27 @@
-{
-  "nio/MSSQLDelete": {
-    "language": "Python",
-    "url": "git://github.com/nio-blocks/mssql.git",
-    "version": "0.2.0"
-  },
-  "nio/MSSQLInsert": {
-    "language": "Python",
-    "url": "git://github.com/nio-blocks/mssql.git",
-    "version": "0.2.0"
-  },
-  "nio/MSSQLQuery": {
-    "language": "Python",
-    "url": "git://github.com/nio-blocks/mssql.git",
-    "version": "0.2.0"
-  },
-  "nio/MSSQLUpdate": {
-    "language": "Python",
-    "url": "git://github.com/nio-blocks/mssql.git",
-    "version": "0.2.0"
-  }
-}
+{
+  "nio/MSSQLRawQuery": {
+    "language": "Python",
+    "url": "git://github.com/nio-blocks/mssql.git",
+    "version": "1.0.0"
+  },
+  "nio/MSSQLDelete": {
+    "language": "Python",
+    "url": "git://github.com/nio-blocks/mssql.git",
+    "version": "1.0.0"

```

Comment:
Python reflection: https://github.com/nio-blocks/communication/blob/master/release.json#L9

Reply:
done. that's neat.